package.json

```
@@ -46,6 +46,7 @@
     "dev:node": "yarn workspace @midas-capital/sdk hardhat node",
     "dev:fork:chapel:node": "FORK_URL_BSC=${MY_CHAPEL_FORK_URL} FORK_BLOCK_NUMBER= FORK_CHAIN_ID=97 yarn workspace @midas-capital/sdk hardhat node --no-deploy --tags fork",
     "dev:fork:bsc:node": "FORK_URL_BSC=${MY_BSC_FORK_URL} FORK_BLOCK_NUMBER= FORK_CHAIN_ID=56 yarn workspace @midas-capital/sdk hardhat node --no-deploy --tags fork",
+    "dev:fork:polygon:node": "FORK_URL_POLYGON=${MY_POLYGON_FORK_URL} FORK_BLOCK_NUMBER= FORK_CHAIN_ID=137 yarn workspace @midas-capital/sdk hardhat node --no-deploy --tags fork",
```

```suggestion
    "dev:fork:polygon:node": "FORK_URL_BSC=${MY_POLYGON_FORK_URL} FORK_BLOCK_NUMBER= FORK_CHAIN_ID=137 yarn workspace @midas-capital/sdk hardhat node --no-deploy --tags fork",
```

The env variable used throughout hh config and docker is called `FORK_URL_BSC` so it's branded with BSC, which is stems from the first time we introduced fork. 
When we set `FORK_URL_POLYGON` it's not used anywhere. So we need to use  `FORK_URL_BSC` instead.

---

fixed